main/linux-rpi/APKBUILD

```diff
@@ -24,10 +24,13 @@ source="https://kernel.org/pub/linux/kernel/v4.x/linux-$_kernver.tar.xz
 	issue-4973.patch
 
 	config-changes-rpi.armhf
+	config-changes-rpi.armv7
 	config-changes-rpi.aarch64
 	config-changes-rpi2.armhf
-	config-changes-rpi.armv7
 	config-changes-rpi2.armv7
+	config-changes-rpi4.armhf
+	config-changes-rpi4.armv7
+	config-changes-rpi4.aarch64
```

Comment:
Where is the package function for rpi4?

Reply:
Looks like that did it, thanks! Although the error message could've been a bit clearer:
```
>>> linux-rpi-dev: Running split function _dev...
make: Entering directory '/aports/main/linux-rpi/src/linux-4.14'
make[1]: Entering directory '/aports/main/linux-rpi/pkg/linux-rpi-dev/usr/src/linux-headers-4.14.78-0-rpi'
/aports/main/linux-rpi/src/linux-4.14/Makefile:509: arch/aarch64/Makefile: No such file or directory
make[2]: *** No rule to make target 'arch/aarch64/Makefile'.  Stop.
make[1]: *** [/aports/main/linux-rpi/src/linux-4.14/Makefile:264: __build_one_by_one] Error 2
make[1]: Leaving directory '/aports/main/linux-rpi/pkg/linux-rpi-dev/usr/src/linux-headers-4.14.78-0-rpi'
make: *** [Makefile:146: sub-make] Error 2
make: Leaving directory '/aports/main/linux-rpi/src/linux-4.14'
>>> ERROR: linux-rpi-dev: _dev failed
>>> ERROR: linux-rpi: prepare_subpackages failed
>>> ERROR: linux-rpi: rootpkg failed
```